### PR TITLE
added configuration file support

### PR DESCRIPTION
**Pull request by NickolaiBeloguzov**
All credentials and some other parameters are now stored in `config.env` file in the same directory as `labs_scrape_and_text.py` script. If this file is missing, is not a file or has its contents corrupted or incomplete, script will reset configuration file and ask user to manually specify all necessary parameters. All methods regarding configuration processing are stored in `configuration_loader.py` module. Also due to configuration file support, I've decided to include `geckodriver` distributives in the bundle (stored in `/bin/` directory). They occupy ~5MB of disk storage space, but simplify deployment significantly. During first launch/configuration reset script will specify path to geckodriver distributive in `config.env` file acording to end user's system architecture (`geckodriver-win32.exe` for 32bit Windows systems and `geckodriver-win64.exe` 64bit systems). Basically you need to just have FireFox browser installed on your machine to run this script.

Tl;dr Added support for configuration file (.env specifically) and moved all credentials there (+ some other parameters), added geckodriver to the bundle to simplify script's deployment process.

### CODE REVIEW

**Review by lukefeilberg — APPROVED** [review]
Looks awesome, great work on this!
